### PR TITLE
[IMP] helpdesk_mgmt: Add priority in portal list and form view

### PR DESCRIPTION
**Pull request by JasminSForgeFlow**

### CODE REVIEW

File: helpdesk_mgmt/views/helpdesk_ticket_templates.xml [hunk 1]
```diff
@@ -11,10 +11,8 @@
                 t-if="page_name == 'ticket' or ticket"
                 t-attf-class="breadcrumb-item #{'active ' if not ticket else ''}"
             >
-                <a
-                    t-if="ticket"
-                    t-attf-href="/my/tickets?{{ keep_query() }}"
-                >Tickets</a>
+                <a t-if="ticket" t-attf-href="/my/tickets?{{ keep_query() }}">Tickets
```
Comment (pedrobaeza) [inline helpdesk_mgmt/views/helpdesk_ticket_templates.xml]: This diff continues. Please review your pre-commit config

Comment (JasminSForgeFlow) [reply helpdesk_mgmt/views/helpdesk_ticket_templates.xml]: I haven't changed any pre-commit config,

Now I push code as it was

**Review by LoisRForgeFlow — APPROVED** [review]
pre-commit changes cleaned now

LGTM :+1:

**Comment by LoisRForgeFlow** [comment]
@pedrobaeza is this good to merge now?

**Comment by pedrobaeza** [comment]
Apart from still some unwanted diff (but it's now very low), I'm doubting this is something adequate to be shown to the customers: it's an internal rating for classifying the tickets, and show such internal criteria to customers may get them angry if doesn't fit with their expectations. If you want this, I would put it under a configuration setting.

**Comment by elvise** [comment]
@JasminSForgeFlow as you already working in the portal, can we take this chance for to fix the UI ?

Ticket (List):
Schermata 2022-09-08 alle 09 56 10

---------------------------------------------------------------------------------------------------------------------------

Task (List):
Schermata 2022-09-08 alle 09 56 25

---------------------------------------------------------------------------------------------------------------------------

Ticket (Form):
 
Schermata 2022-09-08 alle 10 00 24

---------------------------------------------------------------------------------------------------------------------------

Task (Form):
Schermata 2022-09-08 alle 10 00 10

---------------------------------------------------------------------------------------------------------------------------

Looks reasonable ?

**Pull request closed without merging**